test/Tracker1Ltest.java

```diff
@@ -0,0 +1,53 @@
+package components.tracker;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
```

Comment:
Testing is a little thin. Even if methods are trivial, I always recommend a few extra test cases just to be safe. For example, I would do a length of zero (if that's possible), a length of one, and any larger length.